Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1315,159 +1339,108 @@ private void SetBoundingBoxCollider()
             {
                 Bounds bounds = GetTargetBounds();
                 cachedTargetCollider = Target.AddComponent<BoxCollider>();
-                if (boundsMethod == BoundsCalculationMethod.Renderers)
-                {
-                    cachedTargetCollider.center = bounds.center;
-                    cachedTargetCollider.size = bounds.size;
-                }
-                else if (boundsMethod == BoundsCalculationMethod.Colliders)
-                {
-                    // bounds.center is in world space, but cachedTargetCollider.center is in local space
-                    cachedTargetCollider.center = Target.transform.InverseTransformPoint(bounds.center);
-                    cachedTargetCollider.size = Target.transform.InverseTransformSize(bounds.size);
-                }
+
+                cachedTargetCollider.center = bounds.center;
+                cachedTargetCollider.size = bounds.size;
             }
 
-            Vector3 scale = cachedTargetCollider.transform.lossyScale;
-            Vector3 invScale = new Vector3(1.0f / scale[0], 1.0f / scale[1], 1.0f / scale[2]);
-            cachedTargetCollider.size += Vector3.Scale(boxPadding, invScale);
+            if (boxPadding != Vector3.zero)
+            {
+                Vector3 scale = cachedTargetCollider.transform.lossyScale;
```

Comment:
Is it possible to hit divide by zero here?

Reply:
Will check, the code was already there, I just wrapped it checking boxPadding